temporalcli/commands.workflow_trace.go

```diff
@@ -0,0 +1,67 @@
+package temporalcli
+
+import (
+	"fmt"
+	"github.com/fatih/color"
```

Comment:
```suggestion
	"fmt"

	"github.com/fatih/color"
```

Hrmm, usually go fmt would separate these